src/app/modules/ix-table2/interfaces/table-column.interface.ts

```diff
@@ -6,7 +6,7 @@ export abstract class ColumnComponent<T> {
   sortBy?: (row: T) => string | number;
   sortable?: boolean;
 
-  protected row: T;
+  row: T;
```

Comment:
I don't think it should be public. It's not meant to be accessible from outside now. It's better to figure out a way to call `setRow` in tests.